lib/salesforce_bulk/connection.rb

```diff
@@ -48,7 +48,7 @@ def login()
       @server_url = response_parsed['Body'][0]['loginResponse'][0]['result'][0]['serverUrl'][0]
       @instance = parse_instance()
 
-      @@INSTANCE_HOST = @in_sandbox ? "#{@instance}.my.salesforce.com" : "#{@instance}.salesforce.com"
+      @@INSTANCE_HOST = "#{@instance}.salesforce.com"
```

Comment:
Will this break production?  Salesforce urls have the `my.salesforce.com` in it.

Reply:
No, production is the "else" clause and the code remains the same.